src/app/chat/views/message_format.nim

```diff
@@ -5,27 +5,32 @@ proc sectionIdentifier(message: Message): string =
   if message.contentType == ContentType.Group:
     result = "GroupChatMessage"
 
+proc mention(self: ChatMessageList, pubKey: string): string =
+  if self.status.chat.contacts.hasKey(pubKey):
+    result = ens.userNameOrAlias(self.status.chat.contacts[pubKey])
+  else:
+    result = generateAlias(pubKey)
```

Comment:
if else improvement pattern :) :

```suggestion
  if self.status.chat.contacts.hasKey(pubKey):
    return ens.userNameOrAlias(self.status.chat.contacts[pubKey])
  generateAlias(pubKey)
```

or 

```suggestion
  if not self.status.chat.contacts.hasKey(pubKey):
    return generateAlias(pubKey)
  ens.userNameOrAlias(self.status.chat.contacts[pubKey])
```

what is the exception/guard should be in the if, whatever is the expected/normal result should be the normal return